### PR TITLE
[no ticket][risk=no] Puppeteer rework workspace ui test

### PR DESCRIPTION
**Pull request by aweng98**

### CODE REVIEW

File: e2e/tests/workspace/workspace-ui.spec.ts [hunk 1]
```diff
@@ -73,4 +80,36 @@ describe('Workspace ui tests', () => {
     await workspaces.waitForLoad();
     expect(await workspaces.isLoaded()).toBe(true);
   });
+
+  test('Display workspaces by access levels', async () => {
```
Comment (aweng98) [inline e2e/tests/workspace/workspace-ui.spec.ts:84]: Moved from deleted `workspace-filter-display.spec.ts` test file.

File: e2e/tests/workspace/workspace-ui.spec.ts [hunk 2]
```diff
@@ -27,36 +29,41 @@ describe('Workspace ui tests', () => {
...
       } else {
+        expect(size.height).toBeGreaterThan(0);
+        expect(size.width).toBeGreaterThan(0);
         expect(size.height).toEqual(height);
         expect(size.width).toEqual(width);
       }
     }
-  });
-
-  test('Check Workspace cards on the Workspaces page', async () => {
```
Comment (aweng98) [inline e2e/tests/workspace/workspace-ui.spec.ts:36]: Combined `'Check Workspace cards on the Workspaces page'` and `'Workspace cards'` tests. One less login step.

File: e2e/tests/workspace/workspace-ui.spec.ts [hunk 3]
```diff
@@ -27,36 +29,41 @@ describe('Workspace ui tests', () => {
...
 
+    // Check snowman menu options contains expected options
     const snowmanMenu = await card.getSnowmanMenu();
     const links = await snowmanMenu.getAllOptionTexts();
     expect(links).toEqual(expect.arrayContaining(['Share', 'Edit', 'Duplicate', 'Delete']));
+
+    // Check Last Changed time is valid date time
+    const lastChangedTime = Date.parse(await card.getLastChangedTime());
+    // Date.parse returns 'NaN' if string is not a valid time. NaN is never equal to itself.
+    expect(lastChangedTime === lastChangedTime).toBe(true);
```
Comment (aweng98) [inline e2e/tests/workspace/workspace-ui.spec.ts:63]: Add checking for `Last Changed Time`.

**Review by freemabd — APPROVED** [review]
LGTM

**Pull request merged**